aggregating-profiler/src/main/java/ru/fix/aggregating/profiler/engine/CallAggregate.java

```diff
@@ -89,7 +89,7 @@ public void start(AggregatingCall profiledCall, long currentTimestamp) {
         startMaxThroughputPerSecondAcc.call(currentTimestamp, 1);
 
         activeCallsCountSumAdder.increment();
-        if (numberOfActiveCallsToTrackAndKeepBetweenReports.get() > 0) {
```

Comment:
It would be better to rename `numberOfActiveCallsToTrackAndKeepBetweenReports` since right now there is no reporting process is not responsible for cleaning activeCalls  set. 
 Suggestion: `numberOfLongestActiveCallsToTrack`

Reply:
renamed